src/esp/assets/ResourceManager.cpp

```diff
@@ -123,7 +123,15 @@ void ResourceManager::buildImporters() {
 
 }  // buildImporters
 
+bool ResourceManager::getCreateRenderer() {
+  return metadataMediator_->getCreateRenderer();
+}
+
 void ResourceManager::initDefaultPrimAttributes() {
+  if (!getCreateRenderer()) {
```

Comment:
What is the consequence of this? For example, I would expect the `CHECK` in `addPrimitiveToDrawables` to fail. Are we providing good enough error-handling in case a user tries to set createRenderer to false and also tries to use some primitive? @jturner65

Reply:
It seems like `ResourceManager::initDefaultPrimAttributes` is named like this because it used to be necessary for loading in all primitives. But now only `cubeWireframe` is referenced here because it is being used to draw bounding boxes, which somehow makes it a special case. Skipping this method causes bounding boxes to not show up in the viewer, but this shouldn't matter if we're not rendering anything anyways.

IIRC when I tried adding `cubeWireframe` and raycasting at it, the results were still fine. But I can add that part of the test back in.

I don't understand this very well, so I agree that some additional insight from @jturner65 would be helpful here.

Reply:
Initially all primitives (bounding boxes and navmesh) were managed ad hoc in ResourceManager.  When I added support for primitive objects, I made this function to still support bounding boxes using the old mechanism without necessitating a complete redesign of that process.  

I would suggest disposing of the function completely and using the AssetAttributesManager to provide the template for the bounding box, as is stated in the comment below this block. :)  Barring that, it should be fine to avoid this function if rendered bounding boxes are not required.